plonk-book/src/chapter_3.md

```diff
@@ -60,11 +60,9 @@ In order to use these gates only when needed we introduce selector polynomials (
 The gate constraint equation is represented as follow:
 
 $$
-q_{arith} \cdot arith\_eq +
-q_{logic} \cdot  logic\_eq +
-q_{range} \cdot  range\_eq +
-q_{ECfix} \cdot   ECfix\_eq +
-q_{ECvar} \cdot  ECvar\_eq
+q_{arith} \cdot \textit{arith\_eq} + q_{logic} \cdot  \textit{logic\_eq} + q_{range} \cdot  \textit{range\_eq} +
```

Comment:
Does such a selector polynomial really exist, or is it split up into $q_l, q_r, q_o, q_m, q_c$

Reply:
We can consider that `q_arith` is hierarchically at a higher level. 
`q_l, q_r, q_c, q_m, q_o, q_4` could be considered wire selectors in arithmetic gates, and auxiliary values in other custom gates.
On the other hand `q_arith, q_logic, ...` are gate selectors which determine which gate is activated at any point in the circuit.

The explanation should be expanded.

Reply:
Ok will expand the explanation

Reply:
Great, that makes sense. Shall we do this in a separate pull request. I created an issue to keep track of this idea.